src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -93,17 +93,17 @@ def _find_validation_problem(
 
 
 def _row2resource(
-    df_row: pd.Series,
-    details_df: pd.DataFrame,
+    class_info_row: pd.Series,
+    class_property_df: pd.DataFrame,
 ) -> dict[str, Any]:
     """
     Method that reads one row from the "classes" DataFrame,
     opens the corresponding details DataFrame,
     and builds a dict object of the resource.
 
     Args:
-        df_row: row from the "classes" DataFrame
-        details_df: Excel sheet of the individual class
+        class_info_row: row from the "classes" DataFrame
+        class_property_df: Excel sheet of the individual class
```

Comment:
What has "property" to do with the individual class sheet? There are no properties involved. I would rather name it `class_df`. Or if you want to be more specific, `class_df_with_cardinalities`

Reply:
Well the main df contains all the information about the classes except for the cardinalities which always refer to a property. But I am happy to rename it.